examples/twittermap/web/app/views/twittermap/main.scala.html

```diff
@@ -69,6 +70,14 @@
   <script src='@routes.Assets.versioned("javascripts/cache/services.js")'></script>
   <script src='@routes.Assets.versioned("javascripts/mapresultcache/services.js")'></script>
   <script src='@routes.Assets.versioned("javascripts/map/controllers.js")'></script>
+  <script src='@routes.Assets.versioned("javascripts/lib/QuadTree.js")'></script>
+  <script src='@routes.Assets.versioned("javascripts/lib/L.TileLayer.MaskCanvas.js")'></script>
+  <script src='@routes.Assets.versioned("javascripts/lib/Leaflet.Coordinates-0.1.5.min.js")'></script>
+  <!--
+  <script src='@routes.Assets.versioned("lib/leaflet/QuadTree.js")'></script>
+  <script src='@routes.Assets.versioned("lib/leaflet/L.TileLayer.MaskCanvas.js")'></script>
+  <script src='@routes.Assets.versioned("lib/leaflet/Leaflet.Coordinates-0.1.5.min.js")'></script>
+  -->
```

Comment:
Let's remove this commented block?

Reply:
Yes, this is a typo. I will remove them.